javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,77 @@
+let sumbitBtn = document.getElementById("sub");
+let empty = document.getElementById("empty");
+
+
+
+let apiUrl =
+  "http://api.giphy.com/v1/gifs/search?&api_key=UzMWGrXRdD8bLvzF5UnZerMOmKRAVVZI&q="; ;
+let gifNumber = document.getElementById("num");
+let searchWord = document.getElementById("search");
+let imgWrapper = document.querySelector("#imgWrapper")
+
+
+const search = sumbitBtn.onclick=(e)=>{
+ e.preventDefault();
+
+ if (searchWord.value == null || searchWord.value.trim() == "") {
+   empty.innerHTML = "is empty";
+ } else {
+   
+   let newapiUrl = apiUrl + searchWord.value + "&limit=" + gifNumber.value;
+   console.log(newapiUrl);
+   
+   fetch(newapiUrl)
+   
+     .then((respons) => respons.json())
+    
+     .then((result) => {
+       for (let i = 0; i < result.data.length; i++) {
```

Comment:
Instant of using for loop , in java script there is different method to iterate, Also in class Niels shows some other way to iterate, Please check that.